lib/internal/Magento/Framework/View/Page/Config/Renderer.php

```diff
@@ -358,7 +358,7 @@ protected function addDefaultAttributes($contentType, $attributes)
         }
 
         if ($this->canTypeBeFont($contentType)) {
-            return 'rel="preload" as="font" crossorigin="anonymous"';
+            return 'rel="preload" as="font" crossorigin="anonymous" ' . $attributes;
```

Comment:
As I see in https://github.com/magento/magento2/blob/2.4-develop/lib/internal/Magento/Framework/View/Page/Config/Renderer.php#L326-L341 `$attributes` could be `null`